source/testing_counts_1.Rmd

```diff
@@ -464,22 +478,80 @@ discussed earlier for four children and for ten cows. Conventional
 methods are discussed for comparison in @sec-testing-procedures. Here is the
 resampling solution in RESAMPLING STATS:
 
+```{python}
+# set the number of trials
+n_trials = 10000
 
-<!--- Resampling stats file; Matlab syntax closeish -->
+# set the size of each sample
+sample_size = 10
```

Comment:
```suggestion
sample_size = 10

# Probability of any one calf being female.
p_female = 100 / 206
```

See below for where this is used.

If you do go down this route, would need harmonizing wi the text above.

Reply:
Happy to make the changes to the text as well - do you want me to do that?